recipes/vtkm/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+{% set sha256 = "7b25b556e00619ddd0fe44af70c0237fb65cc8fc539bc510466c1bd310c1cee4" %}
+{% set hash_last_commit = "30fdc7f984dfe0ca12c49254da35d3884ddababc" %}
+
+
+package:
+  name: vtkm
+  version: {{ version }}
+
+source:
+  url:  https://gitlab.kitware.com/vtk/vtk-m/repository/v{{ version }}/archive.tar.gz
+  fn:   vtk-m-v{{ version }}-{{ hash_last_commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  features:
+    - vc14 # [win and py>=35]
+    skip: true # [win and py27]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - cmake >=3.3
+    - gcc >=4.8 # [linux]
+    - tbb
+    - boost >=1.52.0
+    - ninja
+    - vc 14 # [win]
+  run:
+    - python
+    - tbb
+    - boost >=1.52.0
+    - vc 14 # [win]
+
+about:
+  home: http://m.vtk.org/index.php/Main_Page
+  license: BSD 3-Clause
```

Comment:
Please add `license_file: LICENSE.txt` and remove the actual license file from the recipe.